pkg/executor/aggfuncs/func_percentile_test.go

```diff
@@ -57,3 +59,12 @@ func TestPercentile(t *testing.T) {
 		require.Equal(t, want, data[index])
 	}
 }
+
+func TestFix40463(t *testing.T) {
```

Comment:
And add the sql in https://github.com/pingcap/tidb/issues/40463 to ft?
https://github.com/pingcap/tidb/tree/master/tests/integrationtest

Reply:
> And add the sql in #40463 to ft? https://github.com/pingcap/tidb/tree/master/tests/integrationtest

okk